app/helpers/buyers/applications_helper.rb

```diff
@@ -41,6 +42,12 @@ def relation_plans_services(provider)
     end.to_json
   end
 
+  def relation_service_and_create_service_plan_path(provider)
```

Comment:
We do not need this helper
Just use the view

Reply:
The reason why using this is to avoid hardcoding the path `admin_service_service_plans_path`, because Ruby cannot be used from within the JS file.